exasol/analytics/query_handler/query/select.py

```diff
@@ -101,56 +99,47 @@ class ModifyQuery(CustomQuery, AuditData):
     def __init__(
         self,
         query_string: str,
-        # Using DBObjectName instead of str, because counting rows of the modified table
-        # requires to use fully_qualified
+        db_object_type: DbObjectType,
         db_object_name: DBObjectName,
-        db_object_type: DbObjectType | str,
-        db_operation_type: DbOperationType | str,
```

Comment:
We should keep this attribute, but limit it to DBOperationType, audit writer will then assign the value to the span typen column

Reply:
OK, I will fix this in next push.